cli/code_scanner.py

```diff
@@ -152,6 +154,35 @@ def pre_commit_scan(context: click.Context, ignored_args: List[str]):
     return scan_documents(context, documents_to_scan, is_git_diff=True)
 
 
+@click.command()
+@click.argument("ignored_args", nargs=-1, type=click.UNPROCESSED)
+@click.pass_context
+def pre_receive_scan(context: click.Context, ignored_args: List[str]):
+    """ Use this command to scan commits on server side before pushing them to the repository """
```

Comment:
```suggestion
    """ Use this command to scan commits on the server side before pushing them to the repository """
```

Reply:
@MaorDavidzon its not just a comment, this is the description we will show when using --help. need more proper text